src/components/Login/Login.jsx

```diff
@@ -1,5 +1,16 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
In Line 1, PR #74 will deal with linting rule by moving this into eslintrc.js, so the can be removed for now (see .eslintrc.js from branch issue-72/user-list-management if you wish to implement that rule to your branch)